src/components/validation/resources/mod.rs

```diff
@@ -142,6 +142,7 @@ fn deserializer_config_to_serializer(config: &DeserializerConfig) -> encoding::S
         // `message` field... but it's close enough for now.
         DeserializerConfig::Bytes => SerializerConfig::Text(TextSerializerConfig::default()),
         DeserializerConfig::Json { .. } => SerializerConfig::Json(JsonSerializerConfig::default()),
+        DeserializerConfig::Protobuf(_) => unimplemented!(),
```

Comment:
@neuronull is this OK?

Reply:
Thanks for the call out. I think this is OK for now. We don't have a serializer for protobuf yet, so there isn't any alternative really. This is similar to the OctetCounting framer in the function below.
When we get to the validation of components that operate over protobuf, we will have to add a task to implement a protobuf external resource. It's possible we'd have to implement a protobuf serializer at that time, to get full coverage of said components.